packages/fxa-settings/src/components/Settings/SettingsSubRow/index.tsx

```diff
@@ -0,0 +1,140 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import classNames from 'classnames';
+import { useFtlMsgResolver } from '../../../models';
+import {
+  AlertFullIcon as AlertIcon,
+  BackupCodesDisabledIcon,
+  BackupCodesIcon,
+  CheckmarkGreenIcon,
+} from '../../Icons';
+import { FtlMsg } from 'fxa-react/lib/utils';
+
+export type SettingsSubrowProps = {
+  ctaGleanId: string;
+  ctaMessage: string;
+  icon: React.ReactNode;
+  idPrefix: string;
+  isEnabled: boolean;
+  localizedDescription: string;
+  message: React.ReactNode;
+  onCtaClick: () => void;
+  showDescription?: boolean;
+  title: string;
+};
+
+const SettingsSubRow = ({
+  ctaGleanId,
+  ctaMessage,
+  icon,
+  idPrefix,
+  isEnabled,
+  localizedDescription,
+  message,
+  onCtaClick,
+  showDescription,
+  title,
+}: SettingsSubrowProps) => {
+  const StatusIcon = () =>
```

Comment:
Just a nit but to keep the span classes consistent, could do something like this?

```
  <span className="grow-0 shrink-0">
    {isEnabled ? (
      <CheckmarkGreenIcon className="mt-1" mode="enabled" />
    ) : (
      <AlertIcon className="m-0" mode="attention" />
    )}
  </span>
```